### PR TITLE
Format Milvus Helm

### PR DESCRIPTION
**Pull request by ZhifengZhang-CN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Milvus Helm Chart
+
+* For more information about installing and using Helm, see the [Helm Docs](https://helm.sh/docs/). For a quick introduction to Charts, see the [Chart Guide](https://helm.sh/docs/topics/charts/).
+
+* Installs the milvus system [Milvus installation](https://milvus.io/docs/guides/get_started/install_milvus/install_milvus.md)
```
Comment (jielinxu) [inline README.md]: To install Milvus, refer to [Milvus Installation](https://milvus.io/docs/guides/get_started/install_milvus/install_milvus.md).

Comment (ZhifengZhang-CN) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+## Introduction
+This chart bootstraps an milvus deployment on a Kubernetes cluster using the Helm package manager.
```
Comment (jielinxu) [inline README.md]: This chart bootstraps Milvus deployment on a .....

File: README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+$ helm uninstall my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+### Milvus Server Configuration
+
+The following table lists the configurable parameters of the milvus server and their default values.
```
Comment (jielinxu) [inline README.md]: Milvus server

File: README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+| `tolerations`                             | Toleration labels for pod assignment          | `[]`                                                    |
+| `affinity`                                | Affinity settings for pod assignment          | `{}`                                                    |
+| `podAnnotations`                          | Additional pod annotations                    | `{}`                                                    |
+| `podDisruptionBudget.minAvailable`        | Pod disruption minimum available              | `unset`                                                 |
+| `podDisruptionBudget.maxUnavailable`      | Pod disruption maximum unavailable            | `unset`                                                 |
+
+
+### MySQL Configuration
+
+The following table lists the configurable parameters of the mysql chart and their default values.
```
Comment (jielinxu) [inline README.md]: MySQL chart?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+### Milvus Server Configuration
+
+The following table lists the configurable parameters of the milvus server and their default values.
+
+| Parameter                                 | Description                                   | Default                                                 |
+|-------------------------------------------|-----------------------------------------------|---------------------------------------------------------|
+| `primaryPath`                             | Path used to store data and metadata          | `/var/lib/milvus/data`                                  |
+| `timeZone`                                | Time zone                                     | `UTC+8`                                                 |
+| `backendURL`                              | URI format: dialect://username:password@host:port/database,Replace 'dialect' with 'mysql' or 'sqlite' | `""` |
```
Comment (jielinxu) [inline README.md]: Replace "," with "." in front of "Replace".

File: README.md (hunk 4 above)
Comment (ZhifengZhang-CN) [reply README.md]: MySQL is Milvus subchart [(dependencies)](https://helm.sh/docs/topics/chart_best_practices/dependencies/)

File: README.md (hunk 5 above)
Comment (ZhifengZhang-CN) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (ZhifengZhang-CN) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (ZhifengZhang-CN) [reply README.md]: Done

**Pull request merged**